recipes/lib-pod5/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: lib-pod5
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.21
+    - libstdcxx-ng
+
+test:
+  imports:
+    - lib_pod5
+
+about:
```

Comment:
Please add a maintainer for this recipe.

Reply:
Done